.github/CODEOWNERS

```diff
@@ -30,813 +30,810 @@
-#/<NotInRepo>/          @kushagraThapar @TheovanKraay @pjohari-ms
+# ServiceLabel: %Guest Configuration
+# ServiceOwners:                                     @mgreenegit @vivlingaiah
 
-# ServiceLabel: %Customer Insights %Service Attention
-#/<NotInRepo>/          @shefymk
+# ServiceLabel: %HDInsight
+# ServiceOwners:                                     @aim-for-better @idear1203 @deshriva
 
-# ServiceLabel: %Custom Providers %Service Attention
-#/<NotInRepo>/          @manoharp @MSEvanhi
+# ServiceLabel: %HPC Cache
+# ServiceOwners:                                    @romahamu @omzevall
 
-# ServiceLabel: %CycleCloud %Service Attention
-#/<NotInRepo>/          @adriankjohnson
+# AzureSdkOwners: @g2vinay @billwert
+# ServiceLabel: %Azure.Identity
+# PRLabel: %Azure.Identity
+/sdk/identity/                                       @g2vinay @billwert @joshfree @Azure/azure-sdk-write-identity
 
-# ServiceLabel: %Data Bricks %Service Attention
-#/<NotInRepo>/          @arindamc
+# PRLabel: %Image Analysis
+# ServiceLabel: %Image Analysis
+/sdk/vision/azure-ai-vision-imageanalysis/           @dargilco @rhurey
 
-# ServiceLabel: %DataBox %Service Attention
-#/<NotInRepo>/          @tmvishwajit @matdickson @manuaery @madhurinms
+# ServiceLabel: %Image Analysis
+# ServiceOwners:                                     @rhurey @dargilco
 
-# ServiceLabel: %DataBox Edge %Service Attention
-#/<NotInRepo>/          @a-t-mason @ganzee @manuaery
+# ServiceLabel: %Import Export
+# ServiceOwners:                                     @madhurinms
 
-# ServiceLabel: %Data Catalog %Service Attention
-#/<NotInRepo>/          @ingave
+# ServiceLabel: %IoT - CLI
+# ServiceOwners:                                     @Azure/azure-iot-cli-triage
 
-# ServiceLabel: %Data Factory %Service Attention
-#/<NotInRepo>/          @shawnxzq @lmy269 @Jingshu923
+# PRLabel: %KeyVault
+/sdk/keyvault/                                       @vcolin7 @g2vinay @Azure/azsdk-keyvault
 
-# ServiceLabel: %Data Lake %Service Attention
-#/<NotInRepo>/          @sumantmehtams
+# ServiceLabel: %KeyVault
+# AzureSdkOwners:                                    @vcolin7
+# ServiceOwners:                                     @RandalliLama @schaabs @jlichwa
 
-# ServiceLabel: %Data Lake Storage Gen1 %Service Attention
-#/<NotInRepo>/          @sumantmehtams
+# ServiceLabel: %Kubernetes Configuration
+# ServiceOwners:                                     @NarayanThiru
 
-# ServiceLabel: %Data Lake Storage Gen2 %Service Attention
-#/<NotInRepo>/          @sumantmehtams
+# ServiceLabel: %Lab Services
+# ServiceOwners:                                     @Tanmayeekamath
 
-# ServiceLabel: %Data Lake Analytics %Service Attention
-#/<NotInRepo>/          @idear1203
+# PRLabel: %Load Testing
+/sdk/loadtesting/                                    @Harshan01 @prativen
 
-# ServiceLabel: %Data Lake Store %Service Attention
-#/<NotInRepo>/          @sumantmehtams
+/sdk/loganalytics/microsoft-azure-loganalytics/      @ischrei
```

Comment:
Is there intentionally no label for this one?

Reply:
There wasn't one before and I can't figure out what this should be labelled as